app/src/main/res/values/colors.xml

```diff
@@ -5,7 +5,6 @@
   <color name="color_accent">@color/blue1</color>
   <color name="color_error">@color/red1</color>
 
-  <color name="ripple_blue1">@color/blue1_opacity_26</color>
   <color name="ripple_grey1">@color/grey1_opacity_26</color>
```

Comment:
This resource is also unnecessary, right?

Reply:
I am using this for `facilitySelectButton` ripple color. By default text buttons have `colorPrimary` ripple, since it's a toolbar button default ripple is not visible.